### ISSUE TITLE
App exit app alarm clock will not ring again

### ISSUE THREAD

**Issue by yrjwcharm** [emekalites/react-native-alarm-notification#36]

**Comment by akinmac002** [comment]
I have same problem. I'm setting the alarm. I'm closing the application. Some devices do not push notification.

**Comment by yrjwcharm** [comment]
@akinmac002  你那个app 进程杀死还能接受到通知吗。你知道如何处理吗。好像Android 6.0以上不让流氓了。这个问题是不是解决不了了。

**Comment by akinmac002** [comment]
Yes, the problem still exists. Android stopped the background process. An error occurs.

**Comment by akinmac002** [comment]
Problem is solved. My problem was that there was not scheduled alarm in the background on Android 8 and above. Issue: https://developer.android.com/about/versions/oreo/background#migration

Solution:
Edit location:
/node_modules/react-native-alarm-notification/android/src/main/java/com/emekalites/react/alarm/notification/ANAlarmReceiver.java

```
...

public class ANAlarmReceiver extends BroadcastReceiver {
    private static final String TAG = ANAlarmReceiver.class.getSimpleName();

    @Override
    public void onReceive(Context context, Intent intent) {
        Intent service = new Intent(context, ANService.class);
        service.putExtras(intent);

// Add this if and edit startService line:
        if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.O) {
            context.startForegroundService(service);
        } else {
            context.startService(service);
        }
    }
}
```

**Comment by yrjwcharm** [comment]
@akinmac002  大佬。

**Comment by ThanhNghia944** [comment]
Hi @akinmac002 app still crash when kill app 

> Problem is solved. My problem was that there was not scheduled alarm in the background on Android 8 and above. Issue: https://developer.android.com/about/versions/oreo/background#migration
> 
> Solution:
> Edit location:
> /node_modules/react-native-alarm-notification/android/src/main/java/com/emekalites/react/alarm/notification/ANAlarmReceiver.java
> 
> ```
> ...
> 
> public class ANAlarmReceiver extends BroadcastReceiver {
>     private static final String TAG = ANAlarmReceiver.class.getSimpleName();
> 
>     @Override
>     public void onReceive(Context context, Intent intent) {
>         Intent service = new Intent(context, ANService.class);
>         service.putExtras(intent);
> 
> // Add this if and edit startService line:
>         if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.O) {
>             context.startForegroundService(service);
>         } else {
>             context.startService(service);
>         }
>     }
> }
> ```
> 
> Furkan Atakan BOZKURT

**Comment by akinmac002** [comment]
Did you try to delete the android build folder and print again after doing the operations?
> Hi @akinmac002 app still crash when kill app

**Comment by emekalites** [comment]
I have published a new version (1.4.0), let me know if it works.
